tests/test_sallysblog.py

```diff
@@ -27,7 +27,10 @@ def test_yields(self):
         self.assertEqual("6", self.harvester_class.yields())
 
     def test_image(self):
-        pass  # todo
+        self.assertEqual(
+            self.harvester_class.image(),
+            "https://img2.storyblok.com/300x300//f/130848/2999x2000/1d3e98afdd/0k1a5313.jpg&w=640&q=100 640w",
```

Comment:
Not sure what to do about it, though - their URLs seem unusual (valid I suppose, although it seems like the query-string part is missing)

Reply:
on a side note: do you want to open PR with v15 too and instruct me specific tasks I can help you with. I'll have more free time in the month to come and want to help! I see that we are to remove the requests dependency and that's a very welcoming update

Reply:
> the image in the test seems to be from another recipe, not the one we scrape.

Oops - yep, I'd been comparing to [another page](https://sallys-blog.de/rezepte/veganes-gemuse-curry-mit-basmatireis) because there seem to be some differences in the image links (although maybe I wasn't comparing the same image-position in each case..)

I'll update with the param-removal code within the next day or two.

> on a side note: do you want to open PR with v15 too and instruct me specific tasks I can help you with. I'll have more free time in the month to come and want to help! I see that we are to remove the requests dependency and that's a very welcoming update

Cool!  Yep, that all sounds good.  I'll have to refresh my memory about some of the `v15` goals, but I agree removal of the `requests` dependency would be nice, and that's a good first task if you'd like to go for that.

Reply:
> Oops - yep, I'd been comparing to [another page](https://sallys-blog.de/rezepte/veganes-gemuse-curry-mit-basmatireis) because there seem to be some differences in the image links (although maybe I wasn't comparing the same image-position in each case..)

Never mind.. I misunderstood.  Wrong image from the correct page (I realized after being puzzled over how the unit tests could have passed).

Reply:
Thanks for fixing that!  (confirmed with the alternate URL)